docs/en/release_notes/release-3.2.md

```diff
@@ -4,6 +4,50 @@ displayed_sidebar: "English"
+
+### Behavior Changes
+
+- `cbo_decimal_cast_string_strict` is used to control how CBO converts data from the DECIMAL type to the STRING type. The default value `true` indicates that the logic built in v2.5.x and later versions prevails and the system implements strict conversion (namely, the system truncates the generated string and fills 0s based on the scale length). The DECIMAL type is not strictly filled in earlier versions, causing different results when comparing the DECIMAL type and the STRING type. [#40619](https://github.com/StarRocks/starrocks/pull/40619)
+- The default value of the Iceberg Catalog parameter `enable_iceberg_metadata_cache` has been changed to `false`. From version v3.2.1 to v3.2.3, this parameter is uniformly set to `true` by default, indicating that Iceberg metadata information is cached by default. In version v3.2.4 and later, if the Iceberg Catalog metastore is HMS, the default value of this parameter is changed to `false`. However, if the metastore is Glue, the default value of this parameter remains `true`. [#41826](https://github.com/StarRocks/starrocks/pull/41826)
+- The user who can refresh materialized views is changed from the `root` user to the user who creates the materialized views. This change does not affect existing materialized views. [#40670](https://github.com/StarRocks/starrocks/pull/40670)  
+- By default, when comparing columns of constant and string types, StarRocks compares them as strings. Users can use the session variable `cbo_eq_base_type` to adjust the rule used for the comparison. For example, users can set `cbo_eq_base_type` to `decimal`, and StarRocks then compares the columns as numeric values. [#40619](https://github.com/StarRocks/starrocks/pull/40619)
+
+### Improvements
+
+- Shared-data StarRocks clusters with an AWS S3 Storage Volume support partitioning data in uniformly prefixed sub-paths of the bucket, improving the read and write efficiency on data files in S3. [#41627](https://github.com/StarRocks/starrocks/pull/41627)
+- StarRocks supports using the parameter `s3_compatible_fs_list` to specify which S3-compatible object storage can be accessed via AWS SDK, and supports using the parameter `fallback_to_hadoop_fs_list` to specify non-S3-compatible object storages that require access via HDFS Schema (this method necessitates the use of vendor-provided JAR packages). [#41123](https://github.com/StarRocks/starrocks/pull/41123)
+- The compatibility with Trino's SQL statement syntax is optimized to support the syntax conversion from the following functions of Trino: `current_catalog`, `current_schema`, `to_char`, `from_hex`, `to_date`, `to_timestamp`, and `index`. [#41217](https://github.com/StarRocks/starrocks/pull/41217) [#41319](https://github.com/StarRocks/starrocks/pull/41319)[ ](https://github.com/StarRocks/starrocks/pull/40837)[#40803](https://github.com/StarRocks/starrocks/pull/40803)
```

Comment:
[ ](https://github.com/StarRocks/starrocks/pull/40837) --- need the PR ID in []

Reply:
- necessitates --> requires
- Optimized compatibility with Trino. Supports syntax conversion from the following Trino functions: `current_catalog`, `current_schema`, `to_char`, `from_hex`, `to_date`, `to_timestamp`, and `index`.